rest/taskana-rest-spring/src/test/java/pro/taskana/task/rest/TaskControllerIntTest.java

```diff
@@ -1964,32 +1965,35 @@ void should_ThrowException_When_UpdatingTaskOwnerOfClaimedTask() {
   class TransferTasks {
     @TestFactory
     Stream<DynamicTest> should_SetTransferFlagDependentOnRequestBody_When_TransferringTask() {
```

Comment:
Can you change the test name so that it includes the owner?

Reply:
Yes I have changed them